packages/mongo/mongo_driver_tests.js

```diff
@@ -0,0 +1,9 @@
+const MongoDB = NpmModuleMongodb;
+
+
+Tinytest.add('mongo_driver - insert Decimal128', function (test) {
```

Comment:
Thanks for adding this test. Instead of creating a new `mongo_driver_tests.js` file however, could you add this test to the existing `mongo_livedata_tests.js` file? That's where other tests like this one live.